### PR TITLE
Query parameter string s

### PR DESCRIPTION
**Pull request by hackaholic**
Query parameter string s can be used for shorting on a specific column in elasticsearch, why to pipe to linux short

### CODE REVIEW

**Comment by javanna** [comment]
hello @hackaholic could you please sign our CLA so we can get your change in ?

**Comment by hackaholic** [comment]
Hi Luca,

I am not aware of CLA, can you guide me how to sign CLA?

Regards

**Comment by dadoonet** [comment]
@hackaholic https://www.elastic.co//contributor-agreement

**Comment by javanna** [comment]
ping @hackaholic we would like to incorporate your change but you need to sign the CLA, otherwise we can't go ahead. Thanks!

**Comment by hackaholic** [comment]
Hi,
I have done the CLA. please look at the attachment.

Regards,
Kumar

**Comment by javanna** [comment]
test this please

**Comment by javanna** [comment]
@elasticmachine update-branch

**Comment by javanna** [comment]
@elasticmachine update branch

**Comment by javanna** [comment]
test this please

File: docs/reference/cat.asciidoc [hunk 1]
```diff
@@ -115,7 +115,7 @@ case is the eighth one.
 
 [source,sh]
 --------------------------------------------------
-% curl '192.168.56.10:9200/_cat/indices?bytes=b' | sort -rnk8
+% curl '192.168.56.10:9200/_cat/indices?v&bytes=b&s=store.size:desc'
```
Comment (javanna) [inline docs/reference/cat.asciidoc]: Small comment: I think you either remove the v argument or you also need to update the following output as it does not contain the verbose output. Makes sense?

**Comment by javanna** [comment]
heya @hackaholic this PR is really close to being merged. I left a small comment as part of my final review, if you could kindly address that I can then go and merge it in. Thanks!

**Comment by jakelandis** [comment]
@hackaholic could you please take a look at the latest review comment ?

Also, have you signed the [CLA](https://www.elastic.co/contributor-agreement) ?

**Comment by martijnvg** [comment]
Hey @hackaholic can you address Luca's comment and sign the CLA? It is important to use the same email address in CLA as in github / git commit.
This PR has been open for a while now and if there is no more activity then we have to close this PR.

**Comment by hackaholic** [comment]
Hi Elastic/Elasticsearch,

Please find the attached signed CLA

Regards Kumar

File: docs/reference/cat.asciidoc (hunk 1 above)
Comment (jrodewig) [inline docs/reference/cat.asciidoc]: Can you also update this sentence in the preceding paragraph?

> Then we'll pipe our output into `sort` using the appropriate column, which in this case is the eighth one.

Here's my recommendation, but please feel free to use your own:

> Then we'll sort our output by the `store.size` column, in descending order.

**Comment by jrodewig** [comment]
Hi @hackaholic 

Thanks for signing the CLA. Do you mind addressing the following comments:

- https://github.com/elastic/elasticsearch/pull/40926/files#r303889185

- https://github.com/elastic/elasticsearch/pull/40926/files#r336219633

After those are addressed, we can merge your changes in.

**Review by jrodewig — CHANGES_REQUESTED** [review]
Hi @hackaholic 

Thanks for signing the CLA. Do you mind addressing the following comments:

- https://github.com/elastic/elasticsearch/pull/40926/files#r303889185

- https://github.com/elastic/elasticsearch/pull/40926/files#r336219633

After those are addressed, we can merge your changes in.

**Comment by jrodewig** [comment]
Closing due to lack of response. Treating as a bug report.

**Pull request closed without merging**